src/components/Book.jsx

```diff
@@ -11,25 +15,70 @@ function Book({ BOOK }) {
     category,
   } = BOOK;
   const dispatch = useDispatch();
+
+  const [percentage, setPercentage] = useState(0);
+  const changeProgress = () => {
+    if (percentage < 100) {
+      setPercentage(percentage + 5);
+    }
+
+    if (percentage === 100) {
+      dispatch(removeBook(item_id));
+    }
+  };
```

Comment:
- [ OPTIONAL] There is no way to pass an input value for the percentage, so the percentage value will always be 0 because you set it as the default value in your `useState.` I would suggest that you use `Math.random` function to get a percentage value each time a book is created. 👍

Reply:
@Gambit142, Thank you for reviewin my code. I did not have access to the full picture, because I was not able to log in to the account. Please take that in consideration.

Reply:
@Boku52500 👋, it is in the project's requirement the deployed project that you can use as a guide for the styling. 👍